index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- Project Header -->
+    <header>
+        <!-- Header toolbar -->
+        <nav>
+            <div class="content header-toolbar-container">
+                <div class="logo-container">
+                    <div class="logo-item">
+                        <h5 class="logo">My Logo</h5>
+                    </div>
+                </div>
+                <div class="menu-container">
+                    <div class="menu-item">
```

Comment:
- [ ] Replace div with a ul tag and for each item use li tag that has a link tag that will redirect to the corresponding section of the page.

Reply:
In this menu, I only have empty divs that represent the hamburger icon made by myself :)

Reply:
Hello @Nicolaswg  Good job at implementing the changes requested by previous reviewer 👍🌝